tests/test_image.h

```diff
@@ -255,5 +256,11 @@ TEST_CASE("[Image] Modifying pixels of an image") {
 			image3->get_pixel(1, 0).is_equal_approx(Color(0, 0, 0, 0)),
 			"flip_y() should not leave old pixels behind.");
 }
+TEST_CASE("[Image] ImageTexture3D::create() with invalid arguments") {
+	ImageTexture3D image;
+	Vector<Ref<Image>> img;
+	Error e = image.create(Image::Format(), 27, -35, 40, false, img);
+	CHECK(e == ERR_CANT_CREATE);
```

Comment:
Testing for expected failure requires the usage of `ERR_PRINT_OFF/ON` macros, because otherwise the test run is going to be polluted with error messages:
```suggestion
	ERR_PRINT_OFF;
	Error e = image.create(Image::Format(), 27, -35, 40, false, img);
	CHECK(e == ERR_CANT_CREATE);
	ERR_PRINT_ON;
```